### PR TITLE
[FSDE-1420] Updated Chi Vue components with 5.0 standards

### PR DESCRIPTION
**Pull request by janibolkvadze**
### Commit summary

- [x] Fixed Pagination issue after using page jumper
- [x] Added support of emission of `chiSelectAll` and `chiDeselectAll` event
- [x] Fixed HTML Select multiple
- [x] Improved sizing of filters to match the 5.0 standards
- [x] Fixed advanced filters' Popover width

@SergioQCL 
https://ctl.atlassian.net/browse/FSDE-1420

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-129/1/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-129/2/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-129/3/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-129/4/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-129/5/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-129/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://064d80f598a92783fcd1b636778749b7466bdc7b-chi-pr.enterprise.ctl.io

**Review by dani-cl-madrid — APPROVED** [review]

File: src/chi-vue/public/index.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
...
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>Chi Vue</title>
-    <link rel="stylesheet" href="https://assets.ctl.io/chi/5.0.0/chi.css">
-    <link rel="icon" type="image/svg+xml" href="https://assets.ctl.io/chi/5.0.0/assets/images/favicon.svg">
-    <link rel="alternate icon" href="https://assets.ctl.io/chi/5.0.0/assets/images/favicon.ico">
-    <script src="https://assets.ctl.io/chi/5.0.0/js/chi.js"></script>
+    <link rel="stylesheet" href="http://localhost:8000/chi-portal.css">
+    <link rel="icon" type="image/svg+xml" href="http://localhost:8000/assets/images/favicon.svg">
+    <link rel="alternate icon" href="http://localhost:8000/assets/images/favicon.ico">
+    <script src="http://localhost:8000/js/chi.js"></script>
```
Comment (SergioQCL) [inline src/chi-vue/public/index.html:11]: Is this correct?

Comment (janibolkvadze) [reply src/chi-vue/public/index.html:11]: Yes, Chi assets on local are served from these URLs.
This is not a production distributed code, just a development one.

Screenshot 2021-08-02 at 15 10 02

**Review by SergioQCL — COMMENTED** [review]
ACK

**Review by SergioQCL — APPROVED** [review]
ACK

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-129/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f62dcf1a1ff7257207095bd5ba5b30022e392d41-chi-pr.enterprise.ctl.io

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-129/8/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ae5a0e378babcb0e503fe2ebc5b7d2f64c92f05d-chi-pr.enterprise.ctl.io

**Comment by jllr** [comment]
Closing this PR as this functionaltity has been implemented

**Pull request closed without merging**